.npmignore

```diff
@@ -0,0 +1,3 @@
+*.blocks/*/*.tests
+*.blocks/*/*.tmpl-specs
+*.blocks/*/*.spec.js
```

Comment:
Looks like we can just wait for a fix on npm side as they are already in progress.